tests/CMakeLists.txt

```diff
@@ -47,14 +47,17 @@ if(AVIF_ENABLE_FUZZTEST OR AVIF_ENABLE_GTEST OR AVIF_BUILD_APPS)
     target_link_libraries(aviftest_helpers_internal avif_apps_internal avif_internal)
 endif()
 
-# Fuzz target without any fuzzing engine dependency. For easy reproduction of oss-fuzz issues.
-add_executable(repro_avif_decode_fuzzer oss-fuzz/avif_decode_fuzzer.cc oss-fuzz/repro_fuzz.cc)
-set_target_properties(repro_avif_decode_fuzzer PROPERTIES LINKER_LANGUAGE "CXX")
-target_link_libraries(repro_avif_decode_fuzzer avif)
-# The test below exists for coverage and as a usage example: repro_avif_decode_fuzzer [reproducer file path]
-add_test(NAME repro_avif_decode_fuzzer COMMAND repro_avif_decode_fuzzer
-                                               ${CMAKE_CURRENT_SOURCE_DIR}/data/color_grid_alpha_nogrid.avif
-)
+if(AVIF_ENABLE_FUZZTEST)
```

Comment:
This change doesn't seem necessary. `repro_avif_decode_fuzzer` isn't a fuzz test. It is a utility program for reproducing a fuzz test error without building a real fuzz test (linked with a fuzzing engine). So it is not clear whether it should be controlled by `AVIF_ENABLE_FUZZTEST`.

Reply:
I changed the build condition to AVIF_LIB_USE_CXX